json-rpc/README.md

```diff
@@ -200,6 +233,17 @@ must be set to the identifier of the snapshot to roll back to. If `rollbackToSna
 performed unconditionally. You have to be careful about landing in a state with the right
 snapshot.
 
+**Precondition.** This method should be called once before calling `nextState`, unless
+the previous call to `assumeTransition` returned with the status `"DISABLED"`.
+
+**Effect.** If `rollbackToSnapshotId >= 0`, the model checker context is rolled back
+to the snapshot with `rollbackToSnapshotId`. Then, the transition with `transitionId`
```

Comment:
I wonder if we should make rollback its own action, just to separate them logically